index.html

```diff
@@ -1790,19 +1793,19 @@
           When an <a>NFC device</a> comes in communication range,
           <ul>
             <li>
-              If <var>options.target</var> has the value <code>"tag"</code>,
+              If <var>target</var> has the value <code>"tag"</code>,
```

Comment:
Good catch. I stand corrected. The algo is rather long to parse on a small screen.